locust/test/test_main.py

```diff
@@ -2681,15 +2681,11 @@ def my_task(self):
 
                 def master_ready():
                     line = master_proc.stderr.readline()
-                    print(f"Master stderr: {line.strip()}")
                     return "All users spawned" in line
 
-                print("Waiting for master to be ready...")
                 try:
                     poll_until(master_ready, timeout=60, sleep_time=0.1)
-                    print("Master is ready")
-                except PollingTimeoutError as e:
-                    print(f"Master readiness polling timed out: {e}")
+                except PollingTimeoutError:
                     raise
```

Comment:
If you're not printing the error any more this is pretty much exactly the same as not having a try-except at all :)